src/snowflake/snowpark/modin/plugin/utils/numpy_to_pandas.py

```diff
@@ -213,8 +213,8 @@ def map_to_bools(inputs: Any) -> Any:
     "exp": NotImplemented,
     "exp2": NotImplemented,
     "log": lambda obj, inputs: obj.apply(sp_func.ln),  # use built-in function
-    "log2": lambda obj, inputs: obj.apply(sp_func.log, base=2),
-    "log10": lambda obj, inputs: obj.apply(sp_func.log, base=10),
+    "log2": lambda obj, inputs: obj.apply(sp_func._log2),
```

Comment:
shouldn't change

Reply:
fix for broken code path in pandas. @sfc-gh-oplaton we can maybe add to umbrella ticket a follow up or file pandas bug.

Reply:
We will still encounter this form of snowpark function in the future. I'm fine with creating a new snowpark function that circumvents it for now; but the ast should handle this eventually.